modules/ListView.js

```diff
@@ -22,7 +22,9 @@
       if (view.__delayedRenderTimeout) {
         clearTimeout(view.__delayedRenderTimeout);
         view.__delayedRenderTimeout = null;
-        view.render();
+        if (!view.isDisposed()) {
```

Comment:
this was a weird bug that started showing up consistently in tests with this commit - the delayed render was being triggered after the view was disposed.

These seemed like reasonable protections (in ListView and View).  I'm guessing it was a race condition that we hit due to large numbers of tests being added.